pkg/agent/cniserver/interface_configuration_windows.go

```diff
@@ -504,3 +518,43 @@ func (ic *ifConfigurator) getInterceptedInterfaces(
 func (ic *ifConfigurator) getOVSInterfaceType() int {
 	return internalOVSInterfaceType
 }
+
+func (ic *ifConfigurator) addPostInterfaceCreateHook(containerID, endpointName string, containerAccess *containerAccessArbitrator, hook postInterfaceCreateHook) error {
+	if containerAccess == nil {
+		return fmt.Errorf("container lock cannot be null")
+	}
+	expectedEP, ok := ic.getEndpoint(endpointName)
+	if !ok {
+		return fmt.Errorf("failed to find HNSEndpoint %s", endpointName)
+	}
+	go func() {
+		ifaceName := fmt.Sprintf("vEthernet (%s)", endpointName)
+		var err error
+		pollErr := wait.PollImmediate(time.Second, 60*time.Second, func() (done bool, err error) {
```

Comment:
If you keep `err` as a return variable, I guess the `err` in the function will be the return value and will not be assigned to the above `err`.